frontend/src/lib/components/heroEmoji/HeroEmoji.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import type {HeroEmojiProps} from './HeroEmoji.type';
+
+  type $$Props = HeroEmojiProps;
+
+  // Merge the necessary classes to possible extra classes defined in the props
+  $$restProps.class = `whitespace-nowrap truncate text-clip text-center font-emoji text-[6.5rem] leading-[6.5rem] ${
+    $$restProps.class ?? ''
+  }`;
+</script>
+
+<!--
+@component
+Used for large emojis acting as illustrations.
+
+The content of the component is the emoji or emojis to be displayed.
+
+The content is hidden from screen readers by default. To override,
+add `aria-hidden="false"` to the tag.
+
+### Properties
+
+- Any valid attributes of a `div` element.
+
+### Slots
+
+- default: the emoji or emojis to be displayed.
+
+### Usage
+
+```tsx
+<HeroEmoji>🚀</HeroEmoji>
+```
+-->
+
+<div aria-hidden="true" {...$$restProps}><slot /></div>
```

Comment:
I may say this needs the `role=img` and perhaps we also need to mention the use of `aria-label` or `aria-labelledby`.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role

Reply:
Good point! I'm adding `role="img"` to reflect that fact, but I would still keep it hidden by default because the intended use is [decorative](https://www.w3.org/WAI/tutorials/images/decorative/) only. I also added a note in the docs clarifying that.

We could ofc add a check that if `aria-hidden` is `false`, an `aria-label` is mandatory, but should we consider those kind of checks all at once or try it here already?